src/app/zkapp_limits/zkapp_limits.ml

```diff
@@ -11,7 +11,7 @@ let main () =
     max_event_elements ;
   printf "max field elements for actions per transaction: %d\n"
     max_action_elements ;
-  printf "All possible zkApp account update combinations" ;
+  printf "All possible zkApp account update combinations: \n" ;
```

Comment:
Don't need the space after the colon

Reply:
Actually it is very helpful as before this line was meddling with table with benchmark outcomes. Now is super easy to detect when results are starting